packages/rich-text-editor/src/components/ToggleIconButton/ToggleIconButton.scss

```diff
@@ -0,0 +1,52 @@
+@import "@kaizen/design-tokens/sass/animation";
+@import "@kaizen/design-tokens/sass/border";
+@import "@kaizen/design-tokens/sass/color";
+@import "@kaizen/design-tokens/sass/spacing";
+
+.button {
```

Comment:
Do we need to change the cursor? I noticed that Slack's RTE buttons use a pointer

Reply:
I'd defer to https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html and just the general fact that browsers don't put pointers on buttons.

Related: https://adamsilver.io/blog/buttons-shouldnt-have-a-hand-cursor/

Reply:
Interesting. Especially since it's much more common to see the pointer on a button on websites than not. I wonder how long it will take for people to follow the convention, considering that the more common pattern is actually to have the pointer (as he also mentioned he associated it himself).

I just read part 2 of the blog too, but I still have a question.

In the case where we have an `<a>` that looks like a button, do we still have the pointer? What about the opposite case where `<button>` looks like a link?

Reply:
In theory, if it does act as a link I would assume so? Though I think that A11y best practice is to have your links visually distinct from your buttons - I'm not 100% sure if it would directly violate a rule (ie: not having your links visually distinct from text) but ideally links have unique styles that are consistent :/

Reply:
Looks like our Button component currently does a pointer regardless, I'd definitely be in favour of changing that.

The button that looks like a link situation feels a bit more icky, but it's not a case we've run into yet (that I'm aware of anyway).

Related: https://github.com/cultureamp/kaizen-design-system/discussions/1868